lib/cucumber/multiline_argument/data_table.rb

```diff
@@ -284,18 +284,22 @@ def map_headers(mappings = {}, &block)
       #     end
       #   end
       #
+      # rubocop:disable Style/OptionalBooleanParameter # the optional boolean parameter is kept for retrocompatibility
       def map_column!(column_name, strict = true, &conversion_proc)
         # TODO: Remove this method for 2.0
```

Comment:
Haha, Always love methods like this, Definitely should dive into removing this maybe>? In v8>?

Reply:
If any objection, agree